CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ set(AVIF_PLATFORM_DEFINITIONS)
 set(AVIF_PLATFORM_INCLUDES)
 set(AVIF_PLATFORM_SYSTEM_INCLUDES)
 set(AVIF_PLATFORM_LIBRARIES)
+set(AVIF_LINK_LIBRARIES)
```

Comment:
Please add a comment on what this variable does: we are starting to have a lot of those ..